test/VoucherHubUpgrade.test.ts

```diff
@@ -0,0 +1,72 @@
+// SPDX-FileCopyrightText: 2025 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { loadFixture } from '@nomicfoundation/hardhat-toolbox/network-helpers';
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { env } from '../config/env';
+import { getDeploymentConfig } from '../deploy/deploy';
+import { mineBlockIfOnLocalFork } from '../scripts/utils/mineBlockIfOnLocalFork';
+import * as voucherHubUtils from '../scripts/voucherHubUtils';
+import { VoucherHub__factory } from '../typechain-types';
+
+describe('VoucherHub upgrade (vNEXT)', function () {
+    before(function () {
+        if (!env.IS_LOCAL_FORK) {
+            this.skip();
+        }
+    });
+
+    async function deployFixture() {
+        const voucherHubERC1967ProxyAddress = await ethers.provider
+            .getNetwork()
+            .then((network) => network.chainId.toString())
+            .then((chainId) => {
+                if (chainId != '134') {
+                    console.error('Bellecour fork network required');
+                    process.exit(1);
+                }
+                return getDeploymentConfig(Number(chainId));
+            })
+            .then((config) => config.voucherHubAddress);
```

Comment:
Maybe it's more straightforward like this:
```suggestion
        if ((await ethers.provider.getNetwork()).chainId !== BigInt(BELLECOUR_CHAIN_ID)) {
            console.error('Bellecour fork network required');
            process.exit(1);
        }
        const voucherHubERC1967ProxyAddress = (await getDeploymentConfig(BELLECOUR_CHAIN_ID)).voucherHubAddress;
```

Reply:
Sure updated https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/58/commits/77ddc5d903664a8308cc8689237bacb9d746c55c